region/maridia/inner-green.json

```diff
@@ -1269,6 +1342,45 @@
                     "HiJump",
                     "canPlayInSand"
                   ]
+                },
+                {
+                  "name": "Jump with Momentum",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "canPlayInSand",
+                    "canTrickyJump",
+                    {"adjacentRunway": {
+                      "fromNode": 2,
+                      "usedTiles": 1,
+                      "physics": ["air"]
+                    }},
+                    "canTrickyUseFrozenEnemies"
+                  ],
+                  "note": [
+                    "Gain momentum in the room to the right, then jump immediately on entry in order to land on the first pillar. Jumping too late will likely put Samus into the pit.",
```

Comment:
The momentum jump is to avoid the evir hit, otherwise a crossRoomJump would be easier.

Reply:
The cross room jump is pretty tight relative to the momentum jump, especially since the hit will likely put samus into the sand.

Reply:
I'm suggesting to note why you should do a momentum jump.